modified_cdc_protocol/operation_types/setup_pcr_plate/protocol.rb

```diff
@@ -0,0 +1,344 @@
+# typed: false
+# frozen_string_literal: true
+
+needs 'Standard Libs/PlanParams'
+needs 'Standard Libs/Debug'
+needs 'Standard Libs/Pipettors'
+needs 'Standard Libs/Units'
+needs 'Standard Libs/AssociationManagement'
+needs 'Standard Libs/LabwareNames'
+
+needs 'Collection Management/CollectionActions'
+
+needs 'Microtiter Plates/MicrotiterPlates'
+
+needs 'Microtiter Plates/PlateLayoutGenerator'
+
+needs 'PCR Libs/PCRComposition'
+
+needs 'Modified CDC Protocol/SetupPCRPlateDebug'
+
+needs 'Diagnostic RT-qPCR/DiagnosticRTqPCRHelper'
+
+PLATE = 'PCR Plate'
+LOT_NUM_KEY = 'Lot Number'
+
+
+class Protocol
+  # Standard Libs
+  include PlanParams
+  include Debug
+  include Pipettors
+  include Units
+  include LabwareNames
+  include AssociationManagement
+
+  include SetupPCRPlateDebug
+
+  include CollectionActions
+
+  include MicrotiterPlates
+
+  include DiagnosticRTqPCRHelper
+
+  RP_COMPOSITION_KEY = 'rp_composition'.to_sym
+  N1_COMPOSITION_KEY = 'n1_composition'.to_sym
+  N2_COMPOSITION_KEY = 'n2_composition'.to_sym
+
+  def default_operation_params
+    {
+      program_name: 'Modified_CDC',
+      layout_method: 'modified_primer_layout_two',
+      group_size: 8 #not needed for the modified CDC protocol
+    }
+  end
+
+  def default_job_params
+    {
+    }
+  end
+
+  def main
+    debug_method = setup_test(operations) if debug
+    @job_params = update_all_params(
+      operations: operations,
+      default_job_params: default_job_params,
+      default_operation_params: default_operation_params
+    )
+    if debug
+      operations.each do |op|
+        op.temporary[:options][:layout_method] = debug_method
+      end
+    end
+
+
+    provision_plates(
+      operations: operations
+    )
+
+    operations.retrieve
+
+    record_lot_numbers(operations: operations)
+
+    prepare_materials(operations: operations)
+
+    sort_compositions(operations: operations)
+
+    assemble_master_mix_plates(operations: operations)
+
+    operations.store
+
+    {}
+  end
+
+  #======= provision plates =======#
+
```

Comment:
Is there a reason not to move these methods from Prepare Master Mix into one of the helper modules?